<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiExo -- Support Application to Run Multiple Wallets</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/run</normaloff>:/icons/run</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0,0,0">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ShapeShift Order ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="orderIdEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>Open In &amp;Browser</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="jsonButton">
        <property name="text">
         <string>Open &amp;JSON</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdButton">
        <property name="text">
         <string>Copy &amp;Cmd</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>SS Receive Address:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="receiveAddrEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="txSendButton">
        <property name="text">
         <string>&amp;Open in Browser</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ethereum Address:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ethEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="ethButton">
        <property name="text">
         <string>Copy E&amp;th Cmd</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>XPub Address:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="xpubEdit"/>
      </item>
      <item>
       <widget class="QComboBox" name="coinTypeCB">
        <item>
         <property name="text">
          <string>bitcoin</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/btc</normaloff>:/icons/btc</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bcash</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/bch</normaloff>:/icons/bch</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dash</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/dash</normaloff>:/icons/dash</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>decred</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/dcr</normaloff>:/icons/dcr</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>litecoin</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/ltc</normaloff>:/icons/ltc</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="xpubButton">
        <property name="text">
         <string>Copy &amp;XPub Cmd</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>&amp;Blockchain.info</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Clear &amp;All</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="action_NewWallet">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/wallet</normaloff>:/icons/wallet</iconset>
   </property>
   <property name="text">
    <string>&amp;New Wallet</string>
   </property>
   <property name="toolTip">
    <string>Create new wallet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_RunWallet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/run</normaloff>:/icons/run</iconset>
   </property>
   <property name="text">
    <string>&amp;Run Wallet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_EditWallet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/edit</normaloff>:/icons/edit</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit Wallet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_DeleteWallet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/delete</normaloff>:/icons/delete</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete Wallet</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/settings</normaloff>:/icons/settings</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/exit</normaloff>:/icons/exit</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_SSOrderId">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/shape-shift</normaloff>:/icons/shape-shift</iconset>
   </property>
   <property name="text">
    <string>ShapeShift &amp;Panel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SSOrderPage">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/shape-shift</normaloff>:/icons/shape-shift</iconset>
   </property>
   <property name="text">
    <string>ShapeShift Order Page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+B</string>
   </property>
  </action>
  <action name="action_SSOrderJSON">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/shape-shift</normaloff>:/icons/shape-shift</iconset>
   </property>
   <property name="text">
    <string>ShapeShift Order JSON</string>
   </property>
   <property name="toolTip">
    <string>Open ShapeShift order JSON in broswer.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>orderIdEdit</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>jsonButton</tabstop>
  <tabstop>receiveAddrEdit</tabstop>
  <tabstop>txSendButton</tabstop>
  <tabstop>xpubEdit</tabstop>
  <tabstop>coinTypeCB</tabstop>
  <tabstop>xpubButton</tabstop>
  <tabstop>ethEdit</tabstop>
  <tabstop>ethButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>orderIdEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sendButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>622</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>receiveAddrEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>txSendButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>759</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ethEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>ethButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>769</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xpubEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>xpubButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
